package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "@google-cloud/tasks",
+  "description": "Cloud Tasks API client for Node.js",
+  "version": "0.1.0",
+  "license": "Apache-2.0",
+  "author": "Google LLC",
+  "engines": {
+    "node": ">=4.0.0"
+  },
+  "repository": "googleapis/nodejs-tasks",
+  "main": "src/index.js",
+  "files": [
+    "protos",
+    "src",
+    "AUTHORS",
+    "COPYING"
+  ],
+  "keywords": [
+    "google apis client",
+    "google api client",
+    "google apis",
+    "google api",
+    "google",
+    "google cloud platform",
+    "google cloud",
+    "cloud",
+    "google cloud-tasks",
+    "cloud-tasks",
+    "Cloud Tasks API"
+  ],
+  "contributors": [
+    "Justin Beckwith <justin.beckwith@gmail.com>"
+  ],
+  "scripts": {
+    "test": "repo-tools test run --cmd npm -- run cover",
+    "generate-scaffolding": "repo-tools generate all",
+    "generate-scaffolding-samples": "repo-tools generate lib_samples_readme -l samples/ --config ../.cloud-repo-tools.json",
+    "cover": "nyc --reporter=lcov mocha --require intelli-espower-loader test/*.js && nyc report",
+    "samples-test": "cd samples/ && npm link ../ && npm test && cd ../",
+    "test-no-cover": "repo-tools test run --cmd mocha -- test/*.js --no-timeouts",
```

Comment:
Please remove `--no-timeouts`, and update the source of this madness :)

Reply:
I have updated the source of this madness. It's still my gist, we must commit it somewhere!